tests/conftest.py

```diff
@@ -73,6 +73,10 @@ def base_app():
         SECURITY_PASSWORD_SALT='TEST_SECURITY_PASSWORD_SALT',
         SECRET_KEY='TEST_SECRET_KEY',
     )
+    # update the application with the configuration provided by the test
```

Comment:
I would prefer not to use the ``@pytest.mark.parametrize`` as it IMHO makes the tests much harder to read and understand.